extras/create_send_notifications.py

```diff
@@ -106,13 +121,13 @@ def main():
     # Change to the correct output directory
     os.chdir(os.path.join(NOTIFICATIONS_BASE_DIR, NOTIFICATION_ARCHIVE_DIR))
 
-    # Build list of CyHy orgs
-    cyhy_org_ids = build_cyhy_org_list(db)
-    logging.debug("Found {} CYHY orgs: {}".format(len(cyhy_org_ids), cyhy_org_ids))
+    # Build list of orgs that should receive notifications
+    notification_org_ids = build_notifications_org_list(db)
```

Comment:
Save notifications_to_delete output from build_notifications_org_list():
```suggestion
    notification_org_ids, notifications_to_delete = build_notifications_org_list(db)
```